package.json

```diff
@@ -136,10 +136,11 @@
     "react-cookie": "^2.1.4",
     "react-final-form": "^6.3.0",
     "react-hot-loader": "^4.3.10",
-    "react-overlays": "^0.8.3",
+    "react-overlays": "3.2.0",
```

Comment:
I see that in stripes-components, react-overlays is set to "react-overlays": "^3.2.0".